index.js

```diff
@@ -59,6 +59,23 @@ class Machine {
     })
   }
 
+  static create(machineName, options, done) {
```

Comment:
Can you make `options` an optional argument? Like this:

```js
static create(machineName, options, done) {
  if (typeof options === 'function') {
    done = options
    options = {}
  }
}
```

Reply:
Apart from the machine name, the caller has to specify the driver to create the machine. So, at least one argument "driver" has to be passed.
If "options" is made optional, then the call certainly fails.
Should we still make it an optional argument? Or can you suggest a better way?

Reply:
How about defaulting to virtualbox? `args.push('--driver', options.driver || 'virtualbox')`

Reply:
Sounds good!
So, the default driver would be "virtualbox"

Reply:
To be honest, I don't know if this is a sensible default. I haven't used Docker (or `node-docker-machine`) in a while. What do you think?

Alternatively we can add a `driver` argument: `create(machineName, driver, options, done)`.

Reply:
I feel that making "virtualbox" the default driver is a good choice. I would like to go with it.

Reply:
@vweevers any other feedback on this?